tests/node-tests.js

```diff
@@ -87,11 +89,15 @@ describe('PapaParse', function() {
 	});
 
 	it('asynchronously parsed streaming CSV should be correctly parsed', function(done) {
+		this.timeout(30000);
```

Comment:
I think adding  --timeout 30000 in package.json "test-node" script might be an option

Reply:
@JohnLeSked this test failed is actually not because of timeout. It because done is never run.
we can safely remove timeout

Reply:
yeah, i knew it never reached the done state, it reached the error state, was just my opinion on timeout concern, good to have the test failed fast